### PR TITLE
iotop: add sudo to all commands

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by bl-ue** [comment]
@Managor will you please update the translations as well?

**Comment by Managor** [comment]
Are there any? When using the "Go to file" on github, it shows no other files than just this.

**Comment by bl-ue** [comment]
Ah, I see! I could've checked myself :smile:

File: pages/common/iotop.md [hunk 1]
```diff
@@ -5,28 +5,28 @@
 
 - Start top-like I/O monitor:
 
-`iotop`
+`sudo iotop`
 
 - Show only processes or threads actually doing I/O:
 
-`iotop -o`
+`sudo iotop -o`
```
Comment (bl-ue) [inline pages/common/iotop.md:12]:
```suggestion
`sudo iotop --only`
```

File: pages/common/iotop.md [hunk 2]
```diff
@@ -5,28 +5,28 @@
...
 
 - Show I/O usage in non-interactive mode:
 
-`iotop -b`
+`sudo iotop -b`
```
Comment (bl-ue) [inline pages/common/iotop.md:16]:
```suggestion
`sudo iotop --batch`
```

File: pages/common/iotop.md [hunk 3]
```diff
@@ -5,28 +5,28 @@
...
 
 - Show only I/O usage of processes(Default is to show all threads):
```
Comment (bl-ue) [inline pages/common/iotop.md:18]: Looks like this got message up at some point ;)
```suggestion
- Show only I/O usage of processes (default is to show all threads):
```

File: pages/common/iotop.md [hunk 4]
```diff
@@ -5,28 +5,28 @@
...
 
-`iotop -P`
+`sudo iotop -P`
 
 - Show I/O usage of given PID(s):
 
-`iotop -p {{PID}}`
+`sudo iotop -p {{PID}}`
```
Comment (bl-ue) [inline pages/common/iotop.md:24]:
```suggestion
`sudo iotop --pid={{PID}}`
```

File: pages/common/iotop.md [hunk 5]
```diff
@@ -5,28 +5,28 @@
...
 
 - Show I/O usage in non-interactive mode:
 
-`iotop -b`
+`sudo iotop -b`
 
 - Show only I/O usage of processes(Default is to show all threads):
 
-`iotop -P`
+`sudo iotop -P`
```
Comment (bl-ue) [inline pages/common/iotop.md:20]:
```suggestion
`sudo iotop --processes`
```

File: pages/common/iotop.md [hunk 6]
```diff
@@ -5,28 +5,28 @@
...
 
 - Show I/O usage of given PID(s):
 
-`iotop -p {{PID}}`
+`sudo iotop -p {{PID}}`
 
 - Show I/O usage of a given user:
 
-`iotop -u {{user}}`
+`sudo iotop -u {{user}}`
```
Comment (bl-ue) [inline pages/common/iotop.md:28]:
```suggestion
`sudo iotop --user={{user}}`
```

File: pages/common/iotop.md [hunk 7]
```diff
@@ -5,28 +5,28 @@
...
 
 - Show accumulated I/O instead of bandwidth:
 
-`iotop -a`
+`sudo iotop -a`
```
Comment (bl-ue) [inline pages/common/iotop.md:32]:
```suggestion
`sudo iotop --accumulated`
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Nice @Managor.  Let's also use the long options here, while we're at it.

**Review by bl-ue — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @Managor!

**Pull request merged**